README.md

```diff
@@ -134,14 +134,6 @@ bitters [options]
 | `bitters help`    | Show help                                             |
 | `bitters version` | Show the version number                               |
 
-## The Bourbon Family
```

Comment:
I guess it makes sense to remove this from Bourbon's README, too, now.

https://github.com/thoughtbot/bourbon#the-bourbon-family

Reply:
Removed in https://github.com/thoughtbot/bourbon/commit/723628a9e5ced732843d20e533f833c88358f831